drupal/sites/default/boinc/modules/boincwork/boincwork.module

```diff
@@ -1041,7 +1041,133 @@ function boincwork_get_output() {
  * Get the project configuration XML; used by client software
  */
 function boincwork_get_project_config() {
-  include_boinc('user/get_project_config.php');
+  require_boinc( array('boinc_db', 'util', 'xml') );
```

Comment:
I presume there's no alternative to reimplement the whole RPC? Doing this means we always need to keep it in sync with BOINC's stock RPC.